disclosure_backend/settings.py

```diff
@@ -1,7 +1,8 @@
 import os
+import os.path as op
```

Comment:
okie. I thought `op` was one of the standard shortenings (like `plt` for `matplotlib.pyplot`), but I'm cool to avoid this in the future and revert this when relevant.